lib/galaxy/config.py

```diff
@@ -349,7 +342,6 @@ def __init__(self, **kwargs):
         self.smtp_password = kwargs.get('smtp_password', None)
         self.smtp_ssl = kwargs.get('smtp_ssl', None)
         self.track_jobs_in_database = string_as_bool(kwargs.get('track_jobs_in_database', 'True'))
-        self.start_job_runners = listify(kwargs.get('start_job_runners', ''))
```

Comment:
The config linting framework has the ability to flag such options - we should remember to document these there.